roles/test_operator/tasks/tempest-tests.yml

```diff
@@ -60,12 +65,17 @@
         list_skipped
 
     - name: Set variable
```

Comment:
This is really a nit :see_no_evil: . Maybe a more specific title for the task: `Set includeList variable`?